packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,8 +202,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
-                    allowedValues: options.map(option => option.value)
+                    transform: value => options.find(option => option.value.toString() === value).label,
+                    allowedValues: options.map(option => option.value.toString())
```

Comment:
@radekmie has the component any way of doing a `toString` of the options for the transform and the allowedValues ? 

I would need some help from you to solve this PR.

thanks